src/seedu/addressbook/commands/Command.java

```diff
@@ -70,4 +70,8 @@ public int getTargetIndex() {
     public void setTargetIndex(int targetIndex) {
         this.targetIndex = targetIndex;
     }
+    
+    public boolean isMutating() {
```

Comment:
Exception is not necessary there to enforce overriding. Think about how to do this elegantly. Hint: just one keyword added to the method signature will do. Also, add the necessary comments for the method to inform others to override this method and what does this method do.